processor/integrations/integrations.go

```diff
@@ -1,11 +1,13 @@
 package integrations
 
 import (
+	"database/sql"
```

Comment:
Let's not do the DB connection from integrations. We can do DB query from envValidator and pass the final value to integrations module from main.go.